### DISCUSSION TITLE
How to use @DatabaseView with left join, i need full example

### DISCUSSION THREAD

**Discussion by ulugbekusmanov** [vitusortner/floor#544] [Q&A]

**Comment by mqus** [comment]
If you don't provide any more details we can't help you. Apart from that, asking nicely will probably also improve your chances at getting help. Your current wording ("I need full example") sounds very demanding, which, combined with the lack of context makes for a very low-effort issue. 

Because this is now more than 4 months old and more of a discussion than an actual feature/bug, I will convert this into a discussion :)
